mivs/automated_emails.py

```diff
@@ -92,7 +92,8 @@ def __init__(self, *args, **kwargs):
 
 MIVSEmail(IndieGame, 'Summary of judging feedback for your game', 'reviews_summary.html',
           lambda game: game.status in c.FINAL_GAME_STATUSES and game.reviews_to_email,
-          ident='mivs_reviews_summary')
+          ident='mivs_reviews_summary',
+          post_con=True))
```

Comment:
Looks like there's an extra parentheses here `)`? Maybe a copy/paste error?

If I don't hear back from you, I'll go ahead and fix this and push it out to production.

Reply:
Yes, it's a copy-pasta!